common-arcana.lic

```diff
@@ -587,6 +587,11 @@ module DRCA
     pause 0.5 while UserVars.moons.empty?
   end
 
+  def cast_spell?(data, settings, force_cambrinth = false, cast_lifecycle_lambda = nil)
+    result = cast_spell(data, settings, force_cambrinth, cast_lifecycle_lambda)
+    result ? true : false
```

Comment:
Why not return `cast_spell` directly?

Reply:
Cause ? methods are supposed to return a boolean, but not all paths through cast_spell actually do, so this forces the weirder paths (like rituals?) into a boolean until a better refactor gets done.

Reply:
Makes sense to me, thanks